setup.py

```diff
@@ -16,6 +16,7 @@
     'itsdangerous',
     'requests',
     'PyJWT',
+    'django-crispy-forms',
```

Comment:
Dit is dus wel een dependency voor elk van de projecten die lizard-auth-client gebruiken!